core/src/main/java/org/apache/gravitino/catalog/SchemaOperationDispatcher.java

```diff
@@ -427,7 +442,7 @@ private EntityCombinedSchema internalLoadSchema(NameIdentifier ident) {
           // For some catalogs like PG, the identifier information is not stored in the schema's
           // metadata, we need to check if this schema is existed in the store, if so we don't
           // need to import.
-          .withImported(isEntityExist(ident, SCHEMA));
```

Comment:
`IsEntityExist` shouldn't be used any more. Maybe you can remove this method.

Reply:
Yeah, I'm doing this.